src/tests/test_utils/functional_test_utils/layer_tests_summary/run_parallel.py

```diff
@@ -742,40 +950,48 @@ def __save_log(logs_dir, dir, test_name):
                                 for unique_model in meta_root.find("models"):
                                     for path in unique_model:
                                         for unique_path in path:
-                                            failed_models.add(unique_path.attrib["path"])
+                                            failed_models.add(
+                                                unique_path.attrib["path"]
+                                            )
                 # api conformance has no failed irs
                 if os.path.exists(failed_ir_dir):
-                    output_file_name = failed_ir_dir + '.tar'
+                    output_file_name = failed_ir_dir + ".tar"
                     with tar_open(output_file_name, "w:gz") as tar:
                         tar.add(failed_ir_dir, arcname=os.path.basename(failed_ir_dir))
-                        logger.info(f"All Conformance IRs for failed tests are saved to: {output_file_name}")
+                        logger.info(
+                            f"All Conformance IRs for failed tests are saved to: {output_file_name}"
+                        )
                     rmtree(failed_ir_dir)
                 if len(failed_models) > 0:
-                    with open(failed_models_file_path, "w") as failed_models_file:
-                        failed_models_list = list()
+                    with open(failed_models_file_path, "w", encoding='UTF-8') as failed_models_file:
+                        failed_models_list = []
                         for item in failed_models:
                             failed_models_list.append(f"{item}\n")
                         failed_models_file.writelines(failed_models_list)
                         failed_models_file.close()
         disabled_tests_path = os.path.join(logs_dir, "disabled_tests.log")
-        with open(disabled_tests_path, "w") as disabled_tests_file:
+        with open(disabled_tests_path, "w", encoding='UTF-8') as disabled_tests_file:
             for i in range(len(self._disabled_tests)):
                 self._disabled_tests[i] += "\n"
             disabled_tests_file.writelines(self._disabled_tests)
             disabled_tests_file.close()
             logger.info(f"Disabled test list is saved to: {disabled_tests_path}")
 
         not_run_tests_path = os.path.join(logs_dir, "not_run_tests.log")
-        with open(not_run_tests_path, "w") as not_run_tests_path_file:
+        with open(not_run_tests_path, "w", encoding='UTF-8') as not_run_tests_path_file:
```

Comment:
One more note for us: Move encoding 'UTF-8' to constant?

Reply:
added